### PR TITLE
Feature/139 update delete media

### PR DESCRIPTION
**Pull request by fasbrothers**

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -69,3 +69,9 @@ export const YANDEX_MAP_QUERY = {
 export const YANDEX_MAP_CONTROL = ['zoomControl', 'fullscreenControl'];
 
 export const PROJECT_NAME = 'StayHop';
+
+export const ACCOMMODATION_STEPS = {
```
Comment (gitkoshelew) [inline src/constants/index.ts]: for steps is better to use numeric Enum instead https://www.typescriptlang.org/docs/handbook/enums.html

File: src/pages/accommodations/components/AddMedia.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+function AddMedia({ accommodationId, handleSearchParamsChange }: AddMediaProps) {
+  const [images, setImages] = useState<ImageListType>([]);
+
+  const { mutate, isPending, isSuccess } = useUploadMediaAccommodationMutation({
+    accommodationId,
+    images,
+  });
+
+  useEffect(() => {
+    if (isSuccess) {
```
Comment (gitkoshelew) [inline src/pages/accommodations/components/AddMedia.tsx:23]: make sure that you handle error also

File: src/pages/accommodations/components/AddMedia.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      handleSearchParamsChange(
+        new URLSearchParams({
+          currentStep: ACCOMMODATION_STEPS.amenities,
+          accommodationId: accommodationId,
+        })
+      );
+    }
+  }, [isSuccess, accommodationId, handleSearchParamsChange]);
+
+  return (
```
Comment (gitkoshelew) [inline src/pages/accommodations/components/AddMedia.tsx]: you have 2 contradictory conditions here so you may don't need a Fragment
 if (isPending) return <LoadingPrimary height="60vh" />
return <UploadImage setImages={setImages} images={images} mutate={mutate} isNew={true} />

File: src/pages/accommodations/components/AddMedia.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    <>
+      {isPending && <LoadingPrimary height="60vh" />}
+
+      {!isPending && (
+        <UploadImage setImages={setImages} images={images} mutate={mutate} isNew={true} />
```
Comment (gitkoshelew) [inline src/pages/accommodations/components/AddMedia.tsx]: remove boolean hardcode here 
isNew />
should be enough

**Review by gitkoshelew — APPROVED** [review]

File: src/pages/accommodations/components/AddMedia.tsx (hunk 2 above)
Comment (fasbrothers) [reply src/pages/accommodations/components/AddMedia.tsx:23]: error is handled in Axios interceptors, if there is any error, the toast will appear

**Pull request merged**